draft-ietf-tls-certificate-compression.md

```diff
@@ -118,13 +125,9 @@ compressed_certificate_message
   bytes in the compressed_certificate_message field are converted into the
   Certificate message.
 
-A peer is not required to compress their own Certificate messages even if the
-compress_certficates extension has been negotiated, but MUST be able to
-decompress a received CompressedCertificate message.
-
-If the negotiated compression algorithm is zlib, then the Certificate message
+If the specified compression algorithm is zlib, then the Certificate message
```

Comment:
The message, or the contents of the message?  Usually when we talk about messages (for instance, in relation to the transcript), the handshake header is involved.  That doesn't really need to be replicated here, so I assume that you want the contents.

Reply:
Good point.  I'll fix this in a separate PR.